wafer/settings.py

```diff
@@ -265,6 +263,9 @@
 # For REGISTRATION_MODE == 'form', the form to present
 WAFER_REGISTRATION_FORM = 'wafer.users.forms.ExampleRegistrationForm'
 
+# Allow registered and anonymous users to see registered users
+WAFER_PUBLIC_ATTENDEE_LIST = False
```

Comment:
Do we really want this to default to False? Keeping the list private doesn't strike me as what most people will want